design/kanister-kopia-integration.md

```diff
@@ -449,6 +449,21 @@ Once the pod is running, the controller executes a set of Kopia CLI commands as
 follows.
 
 > 📝 The examples in this section use S3 for illustration purposes.
+- Create a Kopia repository if it doesnt already exists. Example command to
+create repository is as follows:
+
+```sh
+kopia repository create s3 \
```

Comment:
Note that this would require the users to install Kopia. Alternatively, we can just point them to Kopia documentation